src/main/main.ts

```
@@ -987,3 +987,86 @@ ipcMain.handle("get-presets", () => {
+          { name: "JSON", extensions: ["json"] },
+        ],
+      });
+
+      if (result.canceled || !result.filePath) {
+        return null;
+      }
+
+      const clips = getClips(projectId);
+      const categories = getCategories(projectId);
+      const categoryMap = new Map<number, string>();
+      categories.forEach(cat => categoryMap.set(cat.id!, cat.name));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)